numpy/core/numeric.py

```diff
@@ -2448,6 +2448,10 @@ def allclose(a, b, rtol=1.e-5, atol=1.e-8, equal_nan=False):
     The above equation is not symmetric in `a` and `b`, so that
     `allclose(a, b)` might be different from `allclose(b, a)` in
```

Comment:
Hmm, should be double backticks around `allclose(a, b)`, same for `allclose(b, a)`.  Not yours, but could you fix it?